python/CMakeLists.txt

```diff
@@ -56,5 +56,7 @@ endif()
 include(rapids-cython)
 rapids_cython_init()
 
+add_compile_definitions("SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${RMM_LOGGING_LEVEL}")
```

Comment:
Every .pyx file is a target. And those are spread across two subfolders. So I felt this was easier, and since we want the same logging for all of RMM, it makes sense I think. Do you think I've missed something?

Reply:
That's helpful context. I don't have any great ideas here but @vyasr might.

Reply:
I do think this is appropriate. Because we basically want any RMM Cython to define this, so that it is defined when it includes RMM C++ headers.

Reply:
I think `add_compile_definitions` is fine. I don't envision a scenario where this definition should look different for different Cython TUs.